platforms/allay/README.md

```diff
@@ -0,0 +1,8 @@
+# Allay platform
+
+## Resource files
+
+- `mapping/biomes_JE_1_21_to_BE_1_21_30.json` obtain from GeyserMC/mappings.
+- `mapping/items_JE_1_21_to_BE_1_21_30.json` obtain from GeyserMC/mappings.
+- `mapping/blocks_JE_1_21_to_BE_1_21_30.json` generated by using GeyserMC/mappings-generator, and it's origin name is `generator_blocks.json`.
+- `je_block_default_states_1_21.json` converted from https://zh.minecraft.wiki/w/Module:Block_state_values.
```

Comment:
The mapping json files should not have version info in the file names. they should just be 'mapping/biomes.json' instead of 'mapping/biomes_JE_1_21_to_BE_1_21_30.json' for example.